lightning_client.go

```diff
@@ -153,6 +153,14 @@ type LightningClient interface {
 		opts ...OpenChannelOption) (
 		*wire.OutPoint, error)
 
+	// OpenChannelStream opens a channel to the specified peer and with the
+	// specified arguments and options. This function returns a stream of
+	// updates.
+	OpenChannelStream(ctx context.Context, peer route.Vertex,
```

Comment:
I think this also requires an entry in the `renames` map of the `macaroon_recipes.go` file. Though I wonder why that test didn't fail here...